benchbuild/project.py

```diff
@@ -30,7 +33,15 @@
 from benchbuild.extensions import compiler
 from benchbuild.extensions import run as ext_run
 from benchbuild.settings import CFG
-from benchbuild.utils import db, run, unionfs, wrapping
+from benchbuild.utils import db, run, unionfs, validators, wrapping
+
+if TYPE_CHECKING:
+    from benchbuild.experiment import Experiment
+    from benchbuild.extensions import Extension
+    from benchbuild.utils.container import Container
+    from plumbum.path.local import LocalPath
```

Comment:
E0401 (import-error): Unable to import 'plumbum.path.local'  <!-- Automatically posted by Lintly -->